action-api/src/main/java/com/powsybl/action/MultipleActionsActionBuilder.java

```diff
@@ -46,7 +46,7 @@ public MultipleActionsActionBuilder withActionBuilders(List<ActionBuilder> actio
     }
 
     @Override
-    public Action build() {
+    public MultipleActionsAction build() {
```

Comment:
Are we sure about that? Maybe ask @EtienneLt

Reply:
Seems in line with the other actions